Service.Core/Network/Public/Include/Network/Connection.hpp

```diff
@@ -42,17 +42,17 @@ class Connection : public std::enable_shared_from_this<Connection>
 {
 public:
     /** @enum OwnerType
-     *  @brief A connection is "owned" by either a server or a client. /
+     *  @brief A connection is "owned" by either a service or a client. /
```

Comment:
The old system for transfers, for unlocking a particular code, depending on the place of code execution. It can be executed both on the frontend and on the backend

Reply:
Done